x/posts/internal/keeper/querier.go

```diff
@@ -45,16 +44,9 @@ func getPostResponse(ctx sdk.Context, keeper Keeper, post types.Post) types.Post
 	// Convert the reactions
 	var reactionsResponses = make([]types.ReactionQueryResponse, len(postReactions))
 	for index, reaction := range postReactions {
-		var reactionValue string
-		if em, err := emoji.LookupEmojiByCode(reaction.Value); err == nil {
-			reactionValue = em.Value
-		} else {
-			registeredReaction, _ := keeper.GetRegisteredReaction(ctx, reaction.Value, post.Subspace)
-			reactionValue = registeredReaction.Value
-		}
 		reactionsResponses[index] = types.ReactionQueryResponse{
```

Comment:
Since the new `PostReaction` type is pretty much equal to the `ReactionQueryResponse` type, I think that we can delete the latter and leave only the `PostReaction` type to be used inside the `PostQueryResponse` object.